clients/admin-ui/src/features/system/SystemInformationForm.tsx

```diff
@@ -174,6 +177,7 @@ const SystemInformationForm = ({
         formikHelpers.resetForm({ values });
         onSuccess(systemBody);
         dispatch(setSuggestions("hiding"));
+        dispatch(setActiveSystem(result.data));
```

Comment:
Setting the response as the active system instead of `systemBody` since only the `created_on` field will only be available on the system response, not the system form fields.

Reply:
That makes sense! I shuffled things around to what we had before but with the correct param for onSuccess